mesonbuild/interpreter/kwargs.py

```diff
@@ -308,3 +309,132 @@ class DoSubproject(ExtractRequired):
     version: T.List[str]
     cmake_options: T.List[str]
     options: T.Optional[CMakeSubprojectOptions]
+
+
+GNU_SYMBOL_VISIBILITY = Literal['', 'default', 'internal', 'hidden', 'protected', 'inlineshidden']
+LINK_LANGUAGE = Literal['c', 'cpp', 'objc', 'objcpp', 'd', 'rust', 'swift', 'cuda', 'fortran', 'vala', 'cs', 'java', 'cython']
```

Comment:
Can this somehow use the list from compilers.clink_langs instead of repeating it?

Reply:
Type annotations don't support programmatically creating annotations.

Reply:
Maybe add a comment telling to keep then sync then?

Reply:
I've done that locally, it'll push out with the next update